pop-os/shop
# Upstream rebase

Updates Pop_Shop to be based on the current elementary AppCenter master. Also restores our minimum-width patches and adds a new "Recently Updated" section to the homepage which lists recently updated applications from Flathub. This improves app discoverability and makes the homepage more dynamic (rather than a static page of Pop_Picks).

The new upstream version appears to be much more reliable when installing and updating applications and packages. It also appears to be more responsive.

QA Testing: 
  * Verify that there are no regressions from current versions
  * Verify that the window can be shrunk to narrow, tileable widths
  * Verify that the Pop_Shop Icon and app name appear as expected

Known Issues:
  * App freezes during initial loading; This also appears in the currently released version.
